src/pages/announcement/index.md

```diff
@@ -0,0 +1,5 @@
+---
+template: 'announcement'
+title: tBTC Stakedrop
+---
+The first stakedrop for tBTC is on June 8, 2020. [Add to Calendar](#)
```

Comment:
I would love to have this change to the banner in its own PR so we can merge the editability and then merge this in when we're ready. What do you think?

Reply:
That works for me! Will change to use the old copy!